restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/AccountsEndpoint.java

```diff
@@ -446,7 +450,9 @@ private Account prepareAccountForUpdate(final Account account, final Multivalued
                 if (!validator.isStrongEnough(password))
                     throw new PasswordTooWeak();
                 final String hash = new Md5Hash(data.getFirst("Password")).toString();
-                result = result.setAuthToken(hash);
+                final List<AuthToken> authToken =new ArrayList<>();
+                authToken.add(new AuthToken(account.getSid(), hash, "default"));
```

Comment:
@arslan70 same here. We need to take care account update with multiple auth tokens. If an existing account has already multiple auth tokens, the method here will replace all auth token with a single one